audit-readiness-checklist.md

```diff
@@ -14,6 +14,19 @@
 - [ ]  Run the code through a spellchecker.
 - [ ]  Run a static analysis tool ([Slither](https://github.com/crytic/slither) preferred, [MythX](https://mythx.io/) is an alternative) on your code and consider what it tells you. They very often raise flags for non-issues, but they can sometimes catch low-hanging fruit, so it's worth doing.
 - [ ]  Have at least one trusted Solidity dev or security person outside your organization sanity check your contracts. If your code is a tire fire and in need of major changes, you want to hear about that early and from a trusted friend (for free) rather than after an expensive audit.
+- [ ]  Ether or tokens cannot be accidentally sent to the address 0x0.
+- [ ]  State is being set before and performing actions.
+- [ ]  Only using modifier if necessary in more than one place.
+- [ ]  All types are being explicitly set (e.g. using uint256 instead of uint).
+- [ ]  Using keccak256 instead of the alias sha3.
+- [ ]  Does not use tx.origin anywhere.
+- [ ]  Use revert instead of throw.
+- [ ]  Avoid using problematic features - If you must, be aware of their many nuances
+  - [ ] send (nuances)
+  - [ ] low level functions (call, delegatecall, callcode, inline assembly)
+  - [ ] var
+  - [ ] avoid arbitrary jumps (ref.)
```

Comment:
These are more "code review" style recommendations, that don't follow the general theme of this checklist. Basically all of the above will be highlighted in `slither`. 

Also, I generally don't believe in `address(0)` checks (sometimes fine though).

Reply:
thanks for your feedback, okay fine.